lib/Importer/ProtobufLoader.cpp

```diff
@@ -65,13 +81,13 @@ ProtobufLoader::getNodeValueByName(llvm::StringRef name) const {
 
 llvm::Expected<Constant *>
 ProtobufLoader::createAndRegisterConstant(llvm::StringRef name,
-                                          const Tensor &tensor) {
+                                          Tensor &&tensor) {
   RETURN_ERR_IF_NOT(
       !hasNodeByName(name),
       llvm::Twine("Creating an already existing node ", name).str());
   // Note: We do not support training from models loaded from protos, so
   // trainable is always set to false here.
-  Constant *node = G_.getParent()->createConstant(name, tensor);
+  Constant *node = G_.getParent()->createConstant(name, std::move(tensor));
```

Comment:
More generic would be to forward tensor:
Constant *node = G_.getParent()->createConstant(name, std::forward<Tensor>(tensor));

Reply:
I don't think `std::forward` is applicable here because `Tensor &&tensor` can only bind to rvalue references.